SmartStorePerformance/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "SmartStorePerformance",
+    "version": "0.0.1",
+    "private": true,
+    "scripts": {
+        "start": "node node_modules/react-native/local-cli/cli.js start",
+        "test": "jest"
+    },
+    "sdkDependencies": {
+        "SalesforceMobileSDK-Android": "https://github.com/forcedotcom/SalesforceMobileSDK-Android.git#dev",
+        "SalesforceMobileSDK-iOS": "https://github.com/forcedotcom/SalesforceMobileSDK-iOS.git#dev"
+    },
+    "dependencies": {
+        "native-base": "^2.5.2",
+        "npm": "^5.6.0",
+        "react": "^16.3.1",
+        "react-native": "0.55.4",
+        "react-native-elements": "^1.0.0-beta5",
+        "react-native-force": "git+https://github.com/forcedotcom/SalesforceMobileSDK-ReactNative.git#dev",
+        "react-native-fs": "^2.10.14",
+        "react-native-navigation": "^2.0.2354",
+        "react-native-picker-select": "^3.1.2",
+        "react-native-vector-icons": "^4.6.0",
```

Comment:
You added fonts for Android but I don't think you need to (because they are available in react-native-vector-icons node module).
Simply do:
https://github.com/wmathurin/SimpleApollo/commit/567f2d79801ff5c0323f3e65ce6217ffc29aa282